src/bb_excel/core.clj

```diff
@@ -138,10 +138,13 @@
               (assoc :x row)
               (assoc :y col))]
     (cond
+      ;; Possible data types well explained here https://stackoverflow.com/a/18346273
```

Comment:
I couldn’t reproduce the case when type `d` is used, so I haven’t added a handler for it yet, but it exists in specification. You can get it from link in comment. Probably dates processed by style-check blocks in your code.